cypress/integration/smoke/dashboard.spec.js

```diff
@@ -1,6 +1,9 @@
 const SAFE = 'rin:0xB5ef359e8eBDAd1cd7695FFEF3f6F6D7d5e79B08'
 
 describe('Dashboard', () => {
+  const safeAppsUrl =
+    Cypress.env('CYPRESS_ENV') === 'prod' ? 'https://apps.gnosis-safe.io' : 'https://safe-apps.dev.gnosisdev.com'
```

Comment:
Can't you just reference `process.env.NODE_ENV`?

Reply:
`process.env` is a reference to your environment. Meaning that in order to have `NODE_ENV` accessible from the e2e files I had to set it in the environment that runs the tests, i.e., on running the local script and on the CI environment (GitHub workflow in this case)

I think it could be possible to achieve the same result with that approach but it is not set out of the box and Cypress documents took me in this other direction -> https://docs.cypress.io/guides/guides/environment-variables

Reply:
Environmental dependent variables, such as the Safe Apps URL, I would move to the relevant environmental configuration files within Cypress then.

I'm not entirely sure how it would affect the app but we also have the `process.env.REACT_APP_ENV` environmental variable that is set according to the environment.

Reply:
The env variable itself is fine, but IMHO checking the full app URL is unnecessary.

Reply:
Hey boys! CYPRESS_ENV came out from a conversation with @DiogoSoaress and it is bond to REACT_APP_ENV which is being set in the workflows [depending on the  branch](https://github.com/safe-global/safe-react/blob/c49db48e7f513eef5dc625b9be280514c7660a11/.github/workflows/deploy.yml#L71). Could be useful for future tests as well.

But i completely agree with Ivan about is not necessary to check the complete URL

Reply:
It's definitely good to have it to tell the app env from the tests.

Reply:
Should we not set `REACT_APP_ENV` instead though? It aligns with our other tests then.

Reply:
Ok, I'll resolve this then.